test_fixtures/fairness/experiment.jsonnet

```diff
@@ -0,0 +1,86 @@
+local transformer_model = "epwalsh/bert-xsmall-dummy";
+local transformer_dim = 20;
+
+{
+  "dataset_reader":{
+    "type": "snli_for_bias",
+    "tokenizer": {
+      "type": "pretrained_transformer",
+      "model_name": transformer_model,
+      "add_special_tokens": false
+    },
+    "token_indexers": {
+      "tokens": {
+        "type": "pretrained_transformer",
+        "model_name": transformer_model,
+        "max_length": 512
+      }
+    }
+  },
+  "train_data_path": "test_fixtures/fairness/snli_train.jsonl",
+  "validation_data_path": "test_fixtures/fairness/snli_dev.jsonl",
+  "test_data_path": "test_fixtures/fairness/snli_test.jsonl",
+  "model": {
+    "type": "allennlp.fairness.bias_mitigator_applicator.BiasMitigatorApplicator", 
```

Comment:
Does it need to be a full path?

Reply:
In this case, yes, because BiasMitigatorApplicator is in allennlp.fairness and FromParams only looks at the modules imported in model's __init__.py. I tried placing BiasMitigatorApplicator in model's __init__.py, but this caused a circular dependency issue for some reason - BiasMitigatorApplicator was using some fairness module classes that hadn't been imported yet, etc.